langchain/prompts/prompt.py

```diff
@@ -118,19 +117,8 @@ def from_file(
         Returns:
             The prompt loaded from the file.
         """
-        if not os.path.isfile(template_file):
-            """If the template file is not an absolute path, or from the package
-            directory, try to find it relative to the caller file."""
-            current_dir = os.path.dirname(os.path.abspath((inspect.stack()[1])[1]))
-            template_file = os.path.join(current_dir, template_file)
```

Comment:
This rolls back relative file checking, which is why the test that checks this fails.  Also will need to update the docs examples.   Is this due to the build process for bundling these? Or do you want to keep paths absolute.

Reply:
makes sense, LGTM